src/content/developers/docs/scaling/sidechains/index.md

```diff
@@ -29,7 +29,7 @@ Multiple projects provide implementations of sidechains that you can integrate i
 
 - [Skale](https://skale.network/)
 - [POA Network](https://www.poa.network/)
-- [xDai](https://www.xdaichain.com/)
+- [Gnosis Chain](https://www.xdaichain.com/)
```

Comment:
```suggestion
- [Gnosis Chain (formerly xDai)](https://www.xdaichain.com/)
```
Just want to add some context around the change for the short term to avoid some confusion